content/workers/platform/routing/routes.md

```diff
@@ -64,46 +59,46 @@ While they look similar to a [regex](https://en.wikipedia.org/wiki/Regular_expre
+- Route pattern matching considers the entire request URL, including the query parameter string.  Since route patterns may not contain query parameters, the only way to have a route pattern match URLs with query parameters is to terminate it with a wildcard, `*`.
 
 - Route patterns are case sensitive, for example, `example.com/Images/*` and `example.com/images/*` are two distinct routes.
 
-A route can be specified without being associated with a Worker; this will act to negate any less specific patterns. For example, consider this pair of route patterns, one with a Workers script and one without:
+A route can be specified without being associated with a Worker; this will act to negate any less specific patterns.  For example, consider this pair of route patterns, one with a Workers script and one without:
 
 ```txt
 *example.com/images/cat.png -> <no script>
 *example.com/images/*       -> worker-script
 ```
 
-In this example, all requests destined for example.com and whose paths are prefixed by `/images/` would be routed to `worker-script`, _except_ for `/images/cat.png`, which would bypass Workers completely. Requests with a path of `/images/cat.png?foo=bar` would be routed to `worker-script`, due to the presence of the query string.
+In this example, all requests destined for example.com and whose paths are prefixed by `/images/` would be routed to `worker-script`, _except_ for `/images/cat.png`, which would bypass Workers completely.  Requests with a path of `/images/cat.png?foo=bar` would be routed to `worker-script`, due to the presence of the query string.
 
 ### Validity
 
 The following set of rules govern route pattern validity.
 
 #### Route patterns must include your zone
 
-If your zone is `example.com`, then the simplest possible route pattern you can have is `example.com`, which would match `http://example.com/` and `https://example.com/`, and nothing else. As with a URL, there is an implied path of `/` if you do not specify one.
+If your zone is `example.com`, then the simplest possible route pattern you can have is `example.com`, which would match `http://example.com/` and `https://example.com/`, and nothing else.  As with a URL, there is an implied path of `/` if you do not specify one.
 
 #### Route patterns may not contain any query parameters
 
 For example, `https://example.com/?anything` is not a valid route pattern.
 
 #### Route patterns may optionally begin with http:// or https://
 
-If you omit a scheme in your route pattern, it will match both `http://` and `https://` URLs. If you include `http://` or `https://`, it will only match HTTP or HTTPS requests, respectively.
+If you omit a scheme in your route pattern, it will match both `http://` and `https://` URLs.  If you include `http://` or `https://`, it will only match HTTP or HTTPS requests, respectively.
```

Comment:
```suggestion
If you omit a scheme in your route pattern, it will match both `http://` and `https://` URLs. If you include `http://` or `https://`, it will only match HTTP or HTTPS requests, respectively.
```